feature/experimental/backup_nhg/otg_tests/backup_nhg_test/backup_nhg_test.go

```diff
@@ -427,15 +441,43 @@ func (a *testArgs) validateAftTelemetry(t *testing.T) {
 
 // validateTrafficFlows verifies that the good flow delivers traffic and the
 // bad flow does not deliver traffic.
-func (a *testArgs) validateTrafficFlows(t *testing.T, good *ondatra.Flow, bad *ondatra.Flow) {
-	a.ate.Traffic().Start(t, good, bad)
+func (a *testArgs) validateTrafficFlows(t *testing.T, ate *ondatra.ATEDevice, config gosnappi.Config, good, bad string) {
+
+	waitOTGARPEntry(t)
+	ate.OTG().StartTraffic(t)
 	time.Sleep(15 * time.Second)
-	a.ate.Traffic().Stop(t)
+	ate.OTG().StopTraffic(t)
 
-	if got := a.ate.Telemetry().Flow(good.Name()).LossPct().Get(t); got > 0 {
-		t.Fatalf("LossPct for flow %s: got %g, want 0", good.Name(), got)
+	otgutils.LogFlowMetrics(t, ate.OTG(), config)
+	otgutils.LogPortMetrics(t, ate.OTG(), config)
+	if got := getLossPct(t, ate, good); got > 0 {
+		t.Errorf("LossPct for flow %s: got %v, want 0", good, got)
 	}
-	if got := a.ate.Telemetry().Flow(bad.Name()).LossPct().Get(t); got < 100 {
-		t.Fatalf("LossPct for flow %s: got %g, want 100", bad.Name(), got)
+	if got := getLossPct(t, ate, bad); got < 100 {
+		t.Errorf("LossPct for flow %s: got %v, want 100", bad, got)
+	}
+
```

Comment:
Use got, want 	variable
       if got, want := getLossPct(t, ate, bad), 100; got < want {
		t.Errorf("LossPct for flow %s: got %v, want %v", bad, got, want)
	}